docs/api/web-frame-main.md

```diff
@@ -0,0 +1,91 @@
+# webFrameMain
+
+> Control web pages and iframes.
+
+Process: [Main](../glossary.md#main-process)
+
+## Methods
+
+These methods can be accessed from the `webFrameMain` module:
+
+```javascript
+const { webFrameMain } = require('electron')
+console.log(webFrameMain)
```

Comment:
I think we can do better with this example :) Perhaps show an example of what you might do with the module?

Reply:
I blame the `webContents` docs 😂

I'll update once we decide on a path forward for static methods.

Reply:
Added usage examples in cf87997